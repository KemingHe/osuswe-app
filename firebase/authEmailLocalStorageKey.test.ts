import AUTH_EMAIL_LOCAL_STORAGE_KEY from '@/firebase/authEmailLocalStorageKey';

describe('The Firebase Auth emaiil local storage key', () => {
  it('matches the latest snapshot', () => {
    expect(AUTH_EMAIL_LOCAL_STORAGE_KEY).toMatchSnapshot();
  });
});
