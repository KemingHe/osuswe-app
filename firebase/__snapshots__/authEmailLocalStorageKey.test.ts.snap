// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`The Firebase Auth emaiil local storage key > matches the latest snapshot 1`] = `"firebase-auth-email"`;
